packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -19,6 +60,13 @@
 @breakpoints {
   .rt-DataListRoot {
     &:where(.rt-r-orientation-vertical) {
+      --data-list-value-margin-top: 0px;
+      --data-list-value-margin-bottom: 0px;
+      --data-list-first-item-label-margin-top: var(--leading-trim-start);
+      --data-list-last-item-label-margin-bottom: 0px;
+      --data-list-first-item-value-margin-top: 0px;
+      --data-list-last-item-value-margin-bottom: var(--leading-trim-end);
```

Comment:
We don't _have_ to have a billion CSS variables here, but given the complexity of those selectors across orientation x trim x first/last-child cases, this seems easier to understand and is better when it comes to the amount of the compiled CSS which `@breakpoints` yields